framework/src/main/java/org/checkerframework/framework/type/TypeFromMemberVisitor.java

```diff
@@ -72,10 +75,30 @@ public AnnotatedTypeMirror visitMethod(MethodTree node, AnnotatedTypeFactory f)
         result.setElement(elt);
 
         ElementAnnotationApplier.apply(result, elt, f);
+        new MethodTypeVarAnnotator().visit(result, f);
 
         return result;
     }
 
+    /**
+     * Annotates uses of method type variables with annotation written explicity on the type
```

Comment:
Isn't an annotation explicitly on a type parameter declaration an annotation on the lower bound?
Why is this needed only for method type parameters?

Reply:
>Isn't an annotation explicitly on a type parameter declaration an annotation on the lower bound?

I meant in both the upper and lower bound locations.  I'll clarify the comment.

>Why is this needed only for method type parameters?

This actually apply to class type parameters, too.  I renamed the class and added a test case.